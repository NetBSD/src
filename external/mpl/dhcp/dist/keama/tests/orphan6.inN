# DHCPv6 orphan reservation config

# empty configs are not accepted by Kea
default-lease-time 1800;

# orphan reservation
host foobar {
	hardware ethernet 00:0B:FD:32:E6:FA;
	option dhcp6.name-servers 2a01:e00::2, 2a01:e00::1;
}
