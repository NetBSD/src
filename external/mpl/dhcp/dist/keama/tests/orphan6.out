{
  # DHCPv6 orphan reservation config
  # empty configs are not accepted by Kea
  "Dhcp6": {
    "valid-lifetime": 1800,
    "host-reservation-identifiers": [
      "hw-address"
    ]
//  /// Orphan reservations
//  /// Kea reservations are per subnet
//  /// Reference Kea #231
//  "reservations": [
//    # orphan reservation
//    {
//      "hostname": "foobar",
//      "hw-address": "00:0b:fd:32:e6:fa",
//      "option-data": [
//        {
//          "space": "dhcp6",
//          "name": "dns-servers",
//          "code": 23,
//          "data": "2a01:e00::2, 2a01:e00::1"
//        }
//      ]
//    }
//  ]
  }
}
